src/inspect_scout/_view/www/package.json

```diff
@@ -4,12 +4,28 @@
   "description": "Inspect Scout viewer for evaluation logs.",
   "license": "MIT",
   "type": "module",
+  "main": "./lib/index.js",
+  "module": "./lib/index.js",
+  "types": "./lib/index.d.ts",
+  "files": [
+    "lib",
+    "lib/styles",
+    "lib/assets"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/meridianlabs-ai/inspect_scout",
+    "directory": "src/inspect_scout/_view/www"
+  },
   "packageManager": "pnpm@10.20.0",
   "scripts": {
     "build": "vite build",
```

Comment:
could we update this to explicitly call `vite build:app` (just so it is explicit what the default does).